crates/cxx-qt-lib-extras/build.rs

```diff
@@ -0,0 +1,50 @@
+// SPDX-FileCopyrightText: 2024 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Laurent Montel <laurent.montel@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use cxx_qt_build::CxxQtBuilder;
+use qt_build_utils::QtBuild;
+
+fn main() {
+    let mut builder = CxxQtBuilder::new();
+
+    println!(
+        "cargo:rustc-cfg=qt_version_major=\"{}\"",
```

Comment:
CxxQtBuilder now always instead sets `cxxqt_qt_version_major`, so this line can be removed and the `QtBuild` import and the dependency on `qt_build_utils` in the `Cargo.toml` ?

Reply:
If we remove qt_build_utils from Cargo.toml we get:
```
-- Rust Target: x86_64-unknown-linux-gnu
error: failed to load manifest for workspace member `/opt/travail/git/cxx-qt/crates/cxx-qt-lib-extras`

Caused by:
  failed to parse manifest at `/opt/travail/git/cxx-qt/crates/cxx-qt-lib-extras/Cargo.toml`

Caused by:
  feature `link_qt_object_files` includes `qt-build-utils/link_qt_object_files`, but `qt-build-utils` is not a dependency
CMake Error at /opt/kde-source/kde6-install/share/cmake/CorrosionGenerator.cmake:24 (execute_process):
  execute_process failed command indexes:

    1: "Child return code: 101"

Call Stack (most recent call first):
  /opt/kde-source/kde6-install/share/cmake/CorrosionGenerator.cmake:217 (_cargo_metadata)
  /opt/kde-source/kde6-install/share/cmake/Corrosion.cmake:918 (_generator_add_cargo_targets)
  examples/qml_features/CMakeLists.txt:45 (corrosion_import_crate)

```
=> I keep it.
Or perhaps we can fix it in other toml file

Reply:
Can you change the feature line to be `["cxx-qt-build/link_qt_object_files"]` then this should work